core/src/main/java/org/apache/iceberg/ManifestListWriter.java

```diff
@@ -73,6 +73,7 @@ public long length() {
           .schema(ManifestFile.schema())
           .named("manifest_file")
           .meta(meta)
+          .overwrite(false)
```

Comment:
Let's default manifest lists and manifests to overwrite. These use UUID-based file names and should never conflict.

Reply:
Done.